### PR TITLE
Draft ds-sig charter

### PR DESCRIPTION
**Pull request by MichaelClifford**
This PR includes a draft charter for the DS-SIG that can serve as a starting point for what we want this SIG to be responsible for. It is based heavily on the [DS SIG charter](https://github.com/open-services-group/community/blob/main/sig-data-science/charter.md) in the OSG group. 

I propose we agree on this charter and have it merged first, and in a follow up PR we can update the sigs.yaml and the auto-generated files. 

#174 
#192

### CODE REVIEW

File: sig-data-science/charter.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# SIG Data Science Charter
+
+This charter adheres to the conventions described in the [Operate First Governance document](https://github.com/operate-first/community/blob/main/governance.md) and uses the Roles and Organization Management outlined in that document.
+
+## Scope
+
+SIG Data Science is responsible for all data science and machine learning related initiatives within the Operate First Community. This includes the development, exploration and communication of  cloud native machine learning tools and best practices, MLOps, as well as AIOps tools to enhance the services run by the open services group.
```
Comment (quaid) [inline sig-data-science/charter.md:7]: For this suggestion, I'm also splitting the paragraph into a sentence per line to make the diff easier to read:

```suggestion
SIG Data Science is responsible for all data science and machine learning related initiatives within the Operate First community.
This includes the development, exploration, and communication of cloud native machine learning tools and best practices, MLOps, as well as AIOps tools to enhance the services run by the [Open Services Group](https://github.com/open-services-group/community/).
```

File: sig-data-science/charter.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+* Provide guidance to external groups with onboarding and development of data science work on any of the Operate First managed clusters.
+* Foster a two-way relationship between Operate First and the wider open source data science community at large.
+* Work with other Operate First SIG’s to ensure data collection and reporting is ongoing.
+
+
+### Out of scope
+
+* Not responsible for the installation and availability of data science services maintained by the Operate First Community cloud.
+* Not responsible for any code, notebooks or projects outside of the Operate First Community, and without an associated sub-project initiated by this SIG.
```
Comment (quaid) [inline sig-data-science/charter.md:31]:
```suggestion
* Not responsible for any code, notebooks, or projects outside of the Operate First community, and without an associated subproject initiated by this SIG.
```

File: sig-data-science/charter.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* Not responsible for any request for support on any non-open source data or code.
```
Comment (quaid) [inline sig-data-science/charter.md:32]:
```suggestion
* Not responsible for any request for support on any non-Open Source data or code.
```

File: sig-data-science/charter.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## Roles and Organization Management
+
+This sig adheres to the Roles and Organization Management outlined in [governance document](https://github.com/operate-first/community/blob/main/governance.md)
```
Comment (quaid) [inline sig-data-science/charter.md:36]:
```suggestion
This SIG adheres to the Roles and Organization Management outlined in [governance document](https://github.com/operate-first/community/blob/main/governance.md)
```

**Review by quaid — CHANGES_REQUESTED** [review]
This is great in substance, just some copyedits.

File: sig-data-science/charter.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+SIG Data Science is responsible for all data science and machine learning related initiatives within the Operate First Community. This includes the development, exploration and communication of  cloud native machine learning tools and best practices, MLOps, as well as AIOps tools to enhance the services run by the open services group.
+
+
+### In scope
+
+#### Code, Binaries and Services
+
+* Publish and promotion of videos, blogs, talks and tutorials on the tools and best practices developed by sub-projects.
+* Support the onboarding of new data science projects.
```
Comment (quaid) [inline sig-data-science/charter.md:15]:
```suggestion
* Support the onboarding of new Open Source data science projects.
```

**Comment by MichaelClifford** [comment]
@quaid made the grammar updates you suggested. 

Also, made some larger changes after our discussion in the community SIG meeting last week, including updating the sigs.yaml to include the DS SIG.

**Comment by quaid** [comment]
Holding until we get a second chair and more initial members.
/hold

**Review by aakankshaduggal — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**